src/components/Header/Header.css

```diff
@@ -4,4 +4,24 @@
 }
 
 /* TODO: Add styling for your Header component
-here. */
\ No newline at end of file
+here. */
+
+.todoy {
+    vertical-align: middle;
+    margin-top: 20px;
+    margin-left: 10px;
+}
+
+.todoy-text {
+    vertical-align: bottom;
+    display: inline-block;
+    margin-left: 10px;
+}
+
+.my-profile {
+    float: right;
+    margin-top: 30px;
+    margin-right: 40px;
+    text-decoration: none !important;
```

Comment:
Using `!important` should typically only be used for testing purpose, when it's just more convenient to force a style on a component. Otherwise, if it ends up being in your finalized styles, it probably means that you need to fix the hierarchy for the component(s). In this case, `!important` actually adds nothing to the style, since there's no `text-decoration` styles to override.